jenkins.py

```diff
@@ -104,6 +104,13 @@ def run():
           )
       )
 
+  if (
+      os.getenv('TRAVIS') and
+      (project_dir == 'examples/PocoCpp') and
+      toolchain.startswith('ios-nocodesign')
+  ):
+    verbose = False
```

Comment:
You can add `VERBOSE=0` instead: https://github.com/ingenue/hunter/blob/7cb1740cca46b62739bd7df44a6fa8a5483dec4b/.travis.yml#L65-L66

Reply:
To confirm, you want me to add `VERBOSE=0` in each test `env` inside `.travis.yml` file?

Reply:
Not to every, only for iOS (only for trouble ones).

Reply:
Added, https://github.com/ingenue/hunter/pull/163/commits/4f044ed8ecb272449d6d5d5229b90469e34c9bc8

Now testing ...

Reply:
One of the OSX toolchains failed with
`The job exceeded the maximum time limit for jobs, and has been terminated.` error.

Reply:
> The job exceeded the maximum time limit for jobs

Restarted

Reply:
Please remove this since it's not needed. Then do rebase.